### PR TITLE
fix(watch): watch importmap file for changes

### PR DESCRIPTION
**Pull request by bartOssh**
Fixes [#7519](https://github.com/denoland/deno/issues/7519)

### CODE REVIEW

File: cli/import_map.rs [hunk 1]
```diff
@@ -125,6 +126,17 @@ impl ImportMap {
...
+    if url.scheme() == "file" {
+      return Ok(url.to_file_path().unwrap());
+    }
+    Err(uri_error(format!(
+      "Wrong URL schema: {}. Only 'file' URL schema is allowed.",
+      url.scheme()
+    )))
+  }
+
   fn try_url_like_specifier(specifier: &str, base: &str) -> Option<Url> {
```
Comment (bartlomieju) [inline cli/import_map.rs:140]: This bit is not needed

File: cli/main.rs [hunk 2]
```diff
@@ -483,13 +483,17 @@ async fn run_with_watch(flags: Flags, script: String) -> Result<(), AnyError> {
...
+  let mut paths_to_watch: Vec<PathBuf> = module_graph
     .values()
     .map(|f| Url::parse(&f.url).unwrap())
     .filter(|url| url.scheme() == "file")
     .map(|url| url.to_file_path().unwrap())
     .collect();
 
+  if let Some(import_map) = global_state.maybe_import_map.clone() {
+    paths_to_watch.push(import_map.file_path()?);
+  }
```
Comment (bartlomieju) [inline cli/main.rs:495]: Try using `global_state.flags.import_map_path`

**Review by bartlomieju — CHANGES_REQUESTED** [review]

**Comment by bartOssh** [comment]
revert was a mistake, cleaned

**Review by bartlomieju — APPROVED** [review]
LGTM, thanks @bartOssh

**Pull request merged**